float32 angle
---
bool success
string message
float32 current_angle
